test/webpack.config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     },
     module: {
         loaders: [{
-             test: /\.es6\.js$/, exclude: /node_modules/, loader: 'babel-loader'
+             test: /\.es6\.js$/, exclude: /node_modules/, loader: 'babel-loader?optional[]=runtime'
```

Comment:
A future PR might be nice to DRY this webpack configuration `test/webpack.config.js` and `webpack.config.js`